packages/graphql-language-service/src/parser/getTypeInfo.ts

```diff
@@ -178,6 +178,9 @@ export function getTypeInfo(
               argDefs =
                 directiveDef && (directiveDef.args as GraphQLArgument[]);
               break;
+            case RuleKinds.FRAGMENT_SPREAD:
```

Comment:
Not sure whether getting the type-info is actually needed here

Reply:
it is required, otherwise autocompletion for existing arguments or types will not work. see how `getAutocompleteSuggestions` uses the typeInfo generated here with fragment arguments, for example. You probably don't even need to anything to the AllTypeInfo parser response, as you an re-use the empty `arguments` array. it is a streaming context parser, so typeInfo is re-built for each token

Reply:
so you are lexing properly with the changes in `Rules`, but you still need to track the `arguments` here

Reply:
actually, I think I'm incorrect, because once we reach the Argument tokens, we're dealing with RuleKinds.ARGUMENT already. so I think you are correct, you might not need to change anything here, but I need to look more closely to give a definite answer